roles/pulp/README.md

```diff
@@ -26,9 +29,12 @@ Role Variables:
     See `prereq_pip_packages` also.
 * `pulp_install_api_service`: Whether to create systemd service files for
   pulpcore-api. Defaults to "true".
+* `pulp_update`: Whether to update/upgrade pulpcore to the latest stable release from PyPI within `pulp_branch`. Only affects systems where Pulp is already installed. To limit this to micro (z-stream) updates, make sure to set `pulp_branch`. If `pulpcore_version` is set, or `pulp_source_dir` is set, this has no effect and is effectively always `true`. Defaults to "false".
```

Comment:
I'm leaning towards renaming both this and the plugin's field to "upgrade", because it controls whether or not "--upgrade" is passed to pip (the ansible "pip" module's "state: latest" vs "state: present")